### DISCUSSION TITLE
Input of number type arrows

### DISCUSSION THREAD

**Discussion by AdagioB** [quasarframework/quasar#14863] [Ideas / Proposals] [answered]
When using Qinput as a number, it will show some arrows for the user to quickly increase/decrease the number.
I wish it was possible to set the value it should increase/decrease the number with. Most of the input fields I use are working with decimal numbers, where the users usually just want to increase the number with 0.1 or 0.01. Some of the fields should be increased with steps of 100 or 1000
Would be great if it was also possible to hide the arrows for when it doesn't make sense

**Comment by 0xBiriyani** [comment, marked as answer]
Hey @AdagioB you can use step attribute inside of  q-input to modify rate of change 
step="0.1" or step="10" or you can dynamically bind it to variable
FYI : https://www.w3schools.com/tags/att_input_step.asp

**Reply by AdagioB** [reply]
Perfect, I had been searching for this, but couldn't find anything. Exactly what I was looking for. Thank you so much
